lib/assets/javascripts/constants.js

```diff
@@ -1,2 +1,13 @@
 export const PASSWORD_MIN_LENGTH = 8;
 export const PASSWORD_MAX_LENGTH = 128;
+
```

Comment:
I would create another file for just messages, although this will go away when the translations library is in place...

Reply:
Yeah, lets wait to move these until Jed is in place. If we at least consolidate them all here it will be easier to move I think later on